mirdata/datasets/billboard.py

```diff
@@ -0,0 +1,480 @@
+        return load_sections(
+            os.path.join(self._data_home, self._track_paths["salami"][0]),
+            section_type="letter",
+        )
+
+    @core.cached_property
+    def named_sections(self):
+        return load_sections(
+            os.path.join(self._data_home, self._track_paths["salami"][0]),
+            section_type="name",
+        )
+
+    @core.cached_property
+    def salami_metadata(self):
+        return parse_salami_metadata(
+            os.path.join(self._data_home, self._track_paths["salami"][0])
+        )
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The track's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path, metadata=self._track_metadata
+        )
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
+    """Load a TinySOL audio file.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to audio file
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+
+    """
+    return librosa.load(fhandle, sr=None, mono=True)
+
+
+@io.coerce_to_string_io
+def load_chords(fhandle: TextIO):
+    """Private function to load LAB format chord data from a file
+    Args:
+        chords_path (str):
+    """
+    start_times = []
+    end_times = []
+    chords = []
+
+    reader = csv.reader(fhandle, delimiter="\t")
+    for l in reader:
+        if len(l):
+            start_times.append(float(l[0]))
+            end_times.append(float(l[1]))
+            chords.append(l[2])
+
+    chord_data = annotations.ChordData(np.array([start_times, end_times]).T, chords)
+    return chord_data
+
+
+@io.coerce_to_string_io
+def _parse_timed_sections(fhandle: TextIO) -> List:
+    lines = fhandle.read().split("\n")
+    salami = _parse_salami(lines)
+    assert salami is not None
+    timed_sections = _timed_sections(salami)
+    return timed_sections
+
+
+def load_sections(fpath: str, section_type: str):
+    """Private function to load SALAMI format sections data from a file
+    Args:
+        sections_path (str):
+    """
+
+    timed_sections = _parse_timed_sections(fpath)
+    assert timed_sections is not None
+
+    # Clean sections
+    timed_sections_clean = [ts for ts in timed_sections if ts["section"] is not None]
+
+    start_times = []
+    end_times = []
+    sections = []
+
+    if section_type == "letter":
+        section_label_idx = 0
+    elif section_type == "name":
+        section_label_idx = 1
+    else:
+        raise ValueError("This section type is not available.")
```

Comment:
This line is not being tested. You can add a test by doing something like `load_sections(somepath, "wrong_section_type")`

Reply:
Fixed in https://github.com/Spijkervet/mirdata/commit/b07f3b0153ff21600a796fc3472e5bebc22ead08